Cloud Application Manager/Monitoring/Configure Vars.md

```diff
@@ -0,0 +1,55 @@
+{{{
+  "title": "Configure Vars with Monitoring UI",
+  "date": "6-10-2019",
+  "author": "Thomas Broadwell",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### What are Vars?
+
+Vars are metadata about workloads which are maintained within the Watcher infrastructure.  Vars and Facts, along with Application Lifecycle Manager metadata, are used to make decisions about workloads including the Monitoring Policies to be applied to the workload.
+
+### Viewing Vars
+
+Vars and Facts can be viewed within the Agent details screen of the Monitoring site.  By selecting an agent from the list of all agents, the Agents detail screen is displayed.  
+
+![facts-and-vars-compressed.PNG](../../images/cloud-application-manager/facts-and-vars-compressed.PNG)
```

Comment:
Hey @thomasbroadwell, I would suggest having all your images in a sub-folder into `cloud-application-manager`